honeybee/model.py

```diff
@@ -0,0 +1,480 @@
+# coding: utf-8
+"""Honeybee Model."""
+from .properties import ModelProperties
+from .room import Room
+from .face import Face
+from .shade import Shade
+from .aperture import Aperture
+from .door import Door
+from .boundarycondition import Outdoors, Surface
+from .facetype import AirWall
+from .typing import valid_string
+import honeybee.writer as writer
+
+from ladybug_geometry.geometry3d.face import Face3D
+
+
+class Model(object):
+    """A collection of Rooms, Faces, Shades, Apertures, and Doors representing a model.
+
+    Properties:
+        name
+        name_original
+        rooms
+        faces
+        shades
+        apertures
+        doors
+        has_ngon_apertures
+        had_ngon_doors
+    """
+    __slots__ = ('_name', '_name_original', '_rooms', '_faces', '_shades',
+                 '_apertures', '_doors', '_room_names', '_face_names', '_shade_names',
+                 '_aperture_names', '_door_names', '_properties')
+
+    def __init__(self, name, objects):
+        """A collection of Rooms, Faces, Apertures, and Doors for an entire model.
+
+        Args:
+            name: Model name. Must be < 100 characters.
+            objects: A list of honeybee Rooms, Faces, Apertures, and Doors.
```

Comment:
should model has an input for north?

Reply:
Ah, I got my answer to my comment above.  I'll move the one that I put on honeybee-energy to the honeybee-core one instead.